unitex/build.xml

```diff
@@ -63,8 +63,9 @@
 
 	<target name="dist" depends="clean,compile" description="generate the distribution">
 		<!-- create Unitex.jar -->
-		<jar jarfile="${dist-dir}/Unitex.jar" manifest="classes/fr/umlv/unitex/Manifest.mf" basedir="classes" excludes="classes/fr/loria/**">
+		<jar jarfile="/Users/rojo/Unitex-GramLab-3.1/App/Unitex.jar" manifest="classes/fr/umlv/unitex/Manifest.mf" basedir="classes" excludes="classes/fr/loria/**">
 			<zipgroupfileset dir="${lib-dir}" includes="juniversalchardet-1.0.3.jar"></zipgroupfileset>	
+			<zipgroupfileset dir="${lib-dir}" includes="poi-3.16.jar"></zipgroupfileset>	
```

Comment:
Concerning the use of a third-party library, copyright notices must to be exposed in https://github.com/UnitexGramLab/gramlab-ide/blob/master/LICENSE and the full license must to be distributed in https://github.com/UnitexGramLab/gramlab-ide/tree/master/licenses

Reply:
poi-3.16.jar's licence is include in Apache-2.0